### PR TITLE
Redbot setup documenation

### PR DESCRIPTION
**Pull request by VasilisThePikachu**
Requires https://github.com/space-wizards/wizard-cogs/pull/4 for some images and the watchdog restart cog

### CODE REVIEW

**Comment by VasilisThePikachu** [comment]
wait how the fuck did i...

**Comment by mirrorcult** [comment]
add to summary.md it isnt viewable right now

**Comment by VasilisThePikachu** [comment]
https://github.com/space-wizards/space-station-14/pull/23458 will need me to rewrite part of this probably.

**Comment by VasilisThePikachu** [comment]
Required PR merged.

Ready for review

**Comment by VasilisThePikachu** [comment]
> [space-wizards/space-station-14#23458](https://github.com/space-wizards/space-station-14/pull/23458) will need me to rewrite part of this probably.

I will need for wait for this to get merged anyway and honestly I feel like it will be a bit until then. When the time comes I will update it to match

File: src/en/server-hosting/setting-up-redbot.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+```admonish note title="Psss Combo 2"
+Not an English server? Some cogs have translations, You can change this with ```[p]set locale <language_code>```. Our cogs don't support other languages than English currently.
+```
+Now you are ready, please be sure to check the [Wizard-Cogs Github](https://github.com/space-wizards/wizard-cogs) in case any new cogs have been added.
+
+
+Otherwise, I will list the most interesting ones here with setup instructions
+
+### Game server status
+![SS14 Status](https://github.com/space-wizards/wizard-cogs/raw/master/media/SS14-game-status-example.png)
```
Comment (PJB3005) [inline src/en/server-hosting/setting-up-redbot.md:38]: Copy all these images to the local repo so the links don't break in the future.

Comment (VasilisThePikachu) [reply src/en/server-hosting/setting-up-redbot.md:38]: My idea was that so we don't have to update the images here if we update them on the wizard cogs repo. But I can do when I got time

**Pull request merged**